be/src/simd/simd.h

```diff
@@ -94,4 +94,36 @@ inline size_t count_nonzero(const std::vector<int8_t>& list) {
     return count_nonzero(list.data(), list.size());
 }
 
+inline static size_t find_byte(const std::vector<uint8_t>& list, size_t start, uint8_t byte) {
```

Comment:
Is This similar to`memchr` implementation？

https://gms.tf/stdfind-and-memchr-optimizations.html

Reply:
After benchmark, I found that `memchr` is almost twice times faster than my SIMD implementation. So I will replace it with `memchr`.

```
Run on (104 X 3800 MHz CPU s)
CPU Caches:
  L1 Data 32 KiB (x52)
  L1 Instruction 32 KiB (x52)
  L2 Unified 1024 KiB (x52)
  L3 Unified 36608 KiB (x2)
Load Average: 2.45, 2.48, 2.80
---------------------------------------------------------------------
Benchmark                           Time             CPU   Iterations
---------------------------------------------------------------------
BM_find_zero_plain/8             2.84 ns         2.84 ns    245931732
BM_find_zero_plain/64            26.9 ns         26.9 ns     26097706
BM_find_zero_plain/512            239 ns          239 ns      2945939
BM_find_zero_plain/4096          1852 ns         1852 ns       375966
BM_find_zero_plain/32768        14827 ns        14825 ns        47374
BM_find_zero_simd/8              3.47 ns         3.47 ns    202043696
BM_find_zero_simd/64             12.9 ns         12.9 ns     54648871
BM_find_zero_simd/512            33.7 ns         33.6 ns     21031056
BM_find_zero_simd/4096            183 ns          183 ns      3829427
BM_find_zero_simd/32768          1387 ns         1386 ns       503817
BM_find_zero_stdfind/8           2.10 ns         2.10 ns    333657117
BM_find_zero_stdfind/64          15.0 ns         15.0 ns     46784668
BM_find_zero_stdfind/512          131 ns          131 ns      5337613
BM_find_zero_stdfind/4096         995 ns          995 ns       705406
BM_find_zero_stdfind/32768       7915 ns         7914 ns        88535
BM_find_zero_memchr/8            2.51 ns         2.51 ns    275694325
BM_find_zero_memchr/64           3.46 ns         3.46 ns    202697361
BM_find_zero_memchr/512          13.2 ns         13.2 ns     53007044
BM_find_zero_memchr/4096         81.8 ns         81.8 ns      8574749
BM_find_zero_memchr/32768         651 ns          651 ns      1077890
```